homeassistant/components/trest_solar/strings.json

```diff
@@ -0,0 +1,49 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
```

Comment:
Can we use `data_description` to give more context?

Reply:
Where will this data_description be shown in the GUI? Is it when you are adding a new entry, you get some kind of description?

Reply:
It's a label underneath the input field